configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
some1 can download tarball and re-run autogen/configure sequence and .git/ will not be available.
maybe we can add check AC_PROG_GIT, smth like https://github.com/bdrewery/pkgng/blob/master/configure.ac#L122